### ISSUE TITLE
folder structure when using shared library

### ISSUE THREAD

**Issue by blackfox33** [boredazfcuk/docker-icloudpd#465]
I could not manage to get my desired folder structure for the shared library.
In the old container I got it like this: .../apple/iCloud_Backup_Shared/ and then year/month.
For example: .../apple/iCloud_Backup_Shared/2023/12/file1234.heic

I managed to set Folder Structure: {:%Y/%m} by using libraries_with_dates: true.
But now it does create a subfolder using the name of the shared library, so it looks like that:
.../apple/iCloud_Backup_Shared/SharedSync-{ID_of_the_shared_library}/2023/12/file1234.heic

How can I get rid of the subfolder with the shared library ID and set it to the old structure?

**Comment by rustig136** [comment]
Hi, are you using the 'iCloud Shared Photo Library' and is yours downloading all photos and videos from the shared library?

Thanks!

**Comment by blackfox33** [comment]
Hi, yes I do and it is downloading all photos and videos correctly. It is just this subfolder with the name + ID from the Shared Library that I would like to get rid of and instead download to my desired directory.

**Comment by rustig136** [comment]
Cool, I understand and using a quick network web inspector I could find my ID. Thanks for your prompt reply!

**Comment by rustig136** [comment]
For some reason is my icloudpd not downloading the items from the shared library. I have tried photo_library="all libraries" and "PrimarySync,SharedSync-########-####-####-####-############". So far only my personal library was 100% synced.. Do you want to share your anonymised config with me please?

**Comment by mafis** [comment]
Possible you missing that config: libraries_with_dates: true ? Is default to false and I get into the same problem and read that. @blackfox33 @rustig136

**Comment by rustig136** [comment]
Hello @mafis I appreciate your feedback greatly—it worked wonders. I have to confess that initially, the new setting didn't take effect when I added it to the compose-docker.yml file as an environment variable. However, upon closer inspection of the icloudpd.conf, I noticed "libraries_with_dates:false." This led me to believe that the new setting wasn't applied. After manually modifying the conf file, everything started working as anticipated, and I now have my two libraries in the folder structure as desired.

Thank you very much!
